yt/frontends/amrex/fields.py

```diff
@@ -504,8 +504,8 @@ def setup_fluid_fields(self):
                 )
 
 
-substance_expr_re = re.compile(r"\(([a-zA-Z][a-zA-Z0-9]*)\)")
-substance_elements_re = re.compile(r"(?P<element>[a-zA-Z]+)(?P<digits>\d*)")
+substance_expr_re = re.compile(r"\(([a-zA-Z][a-zA-Z0-9_]*)\)")
```

Comment:
Maybe you guys prefer the verbose style, in which case don't mind this nit, but I couldn't resist pointing out that this expression is now 1-1 equivalent
```suggestion
substance_expr_re = re.compile(r"\(([a-zA-Z]\w*)\)")
```